### PR TITLE
tp-qemu: make iscsi image setup/cleanup via qemu_storage module

### PR DESCRIPTION
**Pull request by xutian**
ID: 1146747

Signed-off-by: Xu Tian <xutian@redhat.com>

### CODE REVIEW

File: qemu/tests/boot_from_device.py [hunk 1]
```diff
@@ -2,8 +2,11 @@
 import re
 import os
 from autotest.client import utils
-from virttest import utils_misc, data_dir, env_process
-from autotest.client.shared import error, iscsi
+from virttest import utils_misc
+from virttest import  data_dir
```
Comment (ypu) [inline qemu/tests/boot_from_device.py]: Seems two blank here.

**Comment by XueqiangWei** [comment]
LGTM,
Acked by Xueqiang Wei <xuwei@redhat.com>

**Comment by PyLearner** [comment]
LGTM,
Acked-by : Meng Yang meyang@redhat.com

**Comment by CongSmile** [comment]
Acked-by: Cong Li <coli@redhat.com>

**Comment by xutian** [comment]
thanks @XueqiangWei @CongSmile  and @PyLearner.

**Pull request merged**